{{cookiecutter.project_slug}}/apps/settings/models.py

```diff
@@ -39,3 +40,47 @@ class FooterHTML(BaseSetting):
 
     class Meta:
         verbose_name = "HTML - Footer"
+
+
+@register_setting(icon="link")
+class SocialAccounts(BaseSetting):
+    facebook_url = models.URLField(
+        blank=True,
+        help_text="""
+        The URL of your organisation\'s Facebook page. E.g.
+        "https://www.facebook.com/organisation_name_or_id"'
+        """,
+    )
+    twitter_handle = models.URLField(
```

Comment:
I think it's better to use twitter url as the URL may change over time

Reply:
I think then we can change that in the code, and have control over the url used. The twitter handle is useful in the twitter meta tags to link the shared posts to the orgs twitter. Do you think we need an extra field for url 🤔